lib/consul/preferences.rb.samples

```diff
@@ -0,0 +1 @@
+SORT_CONSUL_SERVICE_NODES = -> (nodes) { nodes.sort {|a,b| a['Node']['Node'] <=> b['Node']['Node'] } }
```

Comment:
this would not work: we want people to be able to override those settings as part of the template itself!

Reply:
Hmm not sure I understand, what do you mean by that ?

Reply:
lib/consul/preferences.rb is within the code, but users don't have the code, they use the binary consul-templaterb, so, preferences.rb has to be in samples/consul-ui/ itself to allow people tuning it when the software is installed.